README.md

```diff
@@ -81,6 +86,21 @@ envsubst < helm/admin/workflow-pvc.yaml | kubectl apply -n "$NAMESPACE" -f -
 envsubst < helm/admin/secrets.yaml | kubectl apply -n "$NAMESPACE" -f -
 ```
 
+#### Using skaffold
+
+[Skaffold](https://skaffold.dev) can be used to install the OGDC and watch for
+changes in a local environment. Use the `run-local.sh` script to use skaffold:
```

Comment:
Not sure if its worth mentioning, but when I installed with brew it did an outdated version and wasn't able to update using homebrew. May be worth mentioning. Otherwise everything looks great!